applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -898,6 +927,49 @@ pub async fn send_transaction_task(
     }
 }
 
+pub async fn send_one_sided_transaction_task(
```

Comment:
These are gettings quiet long, perhaps move this and the send_transaction_task into a file called tasks.rs in this same module just for neatness sake.